### PR TITLE
Necdet

### PR DESCRIPTION
**Pull request by AsiaBizub**

### CODE REVIEW

**Review by necdetkaskarli1 — APPROVED** [review]

**Review by AsiaBizub — COMMENTED** [review]
Looks good

File: src/test/java/com/trycloud/tests/UserStory7/TestCase1.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+
+public class TestCase1 extends TestBase {
+
+    @Test
+    public void SearchBar(){
```
Comment (aimiebell) [inline src/test/java/com/trycloud/tests/UserStory7/TestCase1.java:15]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Comment (AsiaBizub) [reply src/test/java/com/trycloud/tests/UserStory7/TestCase1.java:15]: Thanks. We are just the beginners. Appreciate your input :)

**Comment by AsiaBizub** [comment]
👍🏻

**Comment by AsiaBizub** [comment]
Looks good.

**Pull request closed without merging**